pkg/userdata/ubuntu/userdata.go

```diff
@@ -390,7 +329,12 @@ write_files:
     [Service]
     Type=oneshot
     RemainAfterExit=true
-    ExecStart=/usr/local/bin/supervise.sh /usr/local/bin/setup
+    ExecStart=/opt/bin/supervise.sh /opt/bin/setup
+
+- path: "/etc/profile.d/opt-bin-path.sh"
+  permissions: "0755"
```

Comment:
This will be sourced and should not be executable, hence have `permissions: "0644"`